guide/src/sextant/pack.md

```diff
@@ -0,0 +1,100 @@
+# Packing an NPK
+
+NPKs are created using the `pack` command of `sextant`.
+The command requires the following input:
+
+1. a manifest file describing the NPK
+2. a root folder containing all files required at runtime
+
+## The Manifest File
+
+The manifest is a YAML file that references all necessary data to mount and execute the container.
+An example `manifest.yaml` of the `hello` container (found in `examples/container/hello/manifest.yaml`) looks as follows:
+
+```yaml
+name: hello
+version: 0.0.1
+init: /hello
+uid: 1000
+gid: 1000
+env:
+  HELLO: northstar
+io:
+  stdout:
+    log:
+      - DEBUG
+      - hello
+mounts:
+    /lib:
+      host: /lib
+    /lib64:
+      host: /lib64
+    /system:
+      host: /system
+```
+
+More details on the manifest format can be found in the chapter
+[The NPK file format](npk_format_details.md).
+
+## The `root` Folder
+
+The root folder contains all the container data that will be available during runtime.
+This includes executable files as well as additional resources.
+During packing, the contents of the folder will be copied to a squashfs image that is then added to the NPK.
+This image file will be mounted by the northstar runtime when the container is run.
+
+## Calling `sextant pack`
+
+Let us `pack` the `hello` example container under Linux.
+First, we build the `hello` binary from the northstar directory using cargo:
+
+```bash
+$ cargo build --release --bin hello
+Compiling hello v0.1.0 (/home/nori/dev/northstar/examples/container/hello)
+Finished release [optimized] target(s) in 2.77s
+```
+
+The resulting binary is found in the `target/release` directory:
+
+```bash
+$ ls target/release/hello
+target/release/hello
+```
+
+Next, we can crate the destination directory if it does not already exist:
+
+```bash
+$ mkdir -p target/northstar/repository
```

Comment:
Maybe it would be good to explain here why this is the chosen path or make some reference to what container repositories are and how containers are "installed"

Reply:
I added a small passage as to why we are choosing that directory. Not sure what to write about 'installing' though.